model/job_queue/v1/job_type.model

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright (c) 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// This struct is a job in a Job Queue.
+class Job {
+    Arguments String // Arguments to run Job with.
```

Comment:
The comments need to go in the line before the definition of the attribute, otherwise they will not be included in the generated OpenAPI documentation.

Reply:
fixed